packages/selleo-design-core/src/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,29 @@
+import { h } from "preact";
+import {
+  DashboardIcon,
+  UsersIcon,
+  BookingDeskIcon,
+  BookingRoomIcon,
+  PropertiesIcon,
+  SelleoLogo,
+} from "../icons";
+import { SidebarItem } from "./SidebarItem";
+
+export function Sidebar() {
+  return (
+    <aside class="w-fit h-full static">
+      <div class="p-3 overflow-y-auto bg-white dark:bg-black relative h-full shadow-[#231f28]/5 shadow-[4px_0_16px]">
```

Comment:
can you check if we can use here the shadows from design system and add them as variables?